.env_sample

```diff
@@ -1,7 +1,2 @@
 Create a file named .env in the root of this directory.  
-Afterwards create the following environment variables:
-
-GMAIL_DOMAIN=YOUR EMAIL DOMAIN HERE
-GMAIL_USERNAME=YOUR EMAIL USERNAME HERE
-GMAIL_PASSWORD=YOUR EMAIL PASSWORD HERE
-SUCCESS_URL=YOUR SUCCESS URL HERE
\ No newline at end of file
+Afterwards create the following environment variables:
```

Comment:
You should leave

`GMAIL_DOMAIN=` 

so that people know what environment variables they need